README.md

```diff
@@ -55,6 +55,17 @@ The easiest way is to fork the [NICE frontend testing base](https://github.com/n
 
 You can use ES6 and [Flow type definitions](https://flow.org/) when developing in the [src folder](src). The source is automatically transpiled with [Babel](https://babeljs.io/docs/usage/cli/) into the lib folder when you [release](#releasing).
 
+### Debug
+
+You can use Given/Then/When/And followed by debug within the feature file which forces a breakpoint:
```

Comment:
Be good to explain here what "forces a breakpoint" means. Probably refer to http://webdriver.io/api/utility/debug.html